extensions/hidemyemail/src/list-emails.tsx

```diff
@@ -1,27 +1,27 @@
-import { useState, useEffect, useRef } from "react";
 import {
-  ActionPanel,
   Action,
-  List,
-  showToast,
-  Toast,
-  Icon,
-  Color,
+  ActionPanel,
+  Alert,
   Clipboard,
+  Color,
+  confirmAlert,
   getPreferenceValues,
+  Icon,
+  Image,
   Keyboard,
-  confirmAlert,
-  Alert,
+  List,
+  showToast,
+  Toast,
   useNavigation,
-  Image,
 } from "@raycast/api";
+import { useCachedPromise, useCachedState } from "@raycast/utils";
+import { useEffect, useRef, useState } from "react";
 import { iCloudService } from "./api/connect";
+import { iCloudSessionExpiredError } from "./api/errors";
 import { getIcon, HideMyEmail, MetaData } from "./api/hide-my-email";
-import { formatTimestamp } from "./utils";
 import { getiCloudService, Login } from "./components/Login";
 import { AddressForm, AddressFormValues } from "./components/forms/AddressForm";
-import { useCachedPromise, useCachedState } from "@raycast/utils";
-import { iCloudSessionExpiredError } from "./api/errors";
+import { formatTimestamp } from "./utils";
```

Comment:
Oh just noticed this. This is just TypeScript sorting imports on save in my editor.